docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -9,6 +9,125 @@
 
 experimental[]
 
+
+Elastic Learned Sparse Encoder - or ELSER - is a representational model trained 
+by Elastic that creates a sparse vector representation of a text. As a retrieval 
+model, ELSER performs text-expansion for more relevant search results with 
+expanding a passage to a sparse representation of tokens that are carefully 
+chosen to improve a semantically relevant retrieval.
+
+You can use ELSER for various NLP tasks, such as 
+{ref}/semantic-search-elser.html[semantic search]. ELSER is an out-of-domain 
+model which doesn't need to be fine-tuned on your data - it provides relevant 
+search results out of the box.
+
+// TO DO: Refer to this blog post, to learn more about ELSER.
+
+
+[discrete]
+[[elser-req]]
+== Requirements
+
+To use ELSER, you must have the {subscriptions}[appropriate subscription] level 
+or the trial period activated.
+
+
+[discrete]
+[[download-elser]]
+== Download ELSER
+
+You can download ELSER either from the **{ml-app}** > **Trained Models** page or 
+by using the Dev Console.
```

Comment:
```suggestion
You can download ELSER either from the **{ml-app}** > **Trained Models** page, 
by using the Dev Console, or using {ents}.
```

Reply:
@droberts195 If I understand [this PR](https://github.com/elastic/kibana/pull/156080) correctly, you need to have ELSER downloaded in order to deploy the model through the Enterprise Search application. This is the reason why I didn't mention Enterprise Search here, only in the section about deploying the model.

Reply:
Okay, I modified in [94d47dd](https://github.com/elastic/stack-docs/pull/2371/commits/94d47dda64547413fa17e280efaa30988499e8cc).

Reply:
> it's possible to both download and deploy ELSER just using the Enterprise Search app right?

Yes, correct.